plugins/com.contrastsecurity.ide.eclipse.ui/src/com/contrastsecurity/ide/eclipse/ui/internal/preferences/OrganizationPreferencesDialog.java

```diff
@@ -1,318 +1,313 @@
-package com.contrastsecurity.ide.eclipse.ui.internal.preferences;
```

Comment:
Did you mean to replace this whole file? Is it the indentation?

Reply:
Indentation problem